core/src/main/resources/hudson/slaves/SlaveComputer/slave-agent.jnlp.jelly

```diff
@@ -89,21 +89,6 @@ THE SOFTWARE.
 
         <argument>-url</argument>
         <argument>${rootURL}</argument>
-
-        <j:if test="${rootURL!=app.rootUrlFromRequest}">
```

Comment:
Too risky for LTS. https://github.com/jenkinsci/jenkins/pull/4880#discussion_r462602812 represents a more conservative fix.

Reply:
For LTS is either backport or not. I am uneasy to incorporate suggested fix that was never merged, released, tested.

Are there other downsides of backporting this except for https://issues.jenkins-ci.org/browse/JENKINS-63222?

Reply:
I agree with Jesse. This isn't a clear improvement. Would leave it as is for this release (and consider expediting and backporting into next.1 the fix proposed by Jesse).

Reply:
I agree, better to not backport it

Reply:
Reverting. Thanks for the feedback